WooCommerce/WooCommerceTests/Extensions/SitePluginsURLTests.swift

```diff
@@ -0,0 +1,46 @@
+import Yosemite
+
+@testable import WooCommerce
+
+import Foundation
+import XCTest
+
+final class Site_PluginsURLTests: XCTestCase {
+    private var adminURL: String!
+    private var site: Site!
+
+    override func setUp() {
+        super.setUp()
+
+        adminURL = "https://testshop.com/wp-admin"
+        site = Site.fake().copy(adminURL: adminURL)
+    }
+
+    override func tearDown() {
+        adminURL = nil
+        site = nil
+
+        super.tearDown()
+    }
+
+    func test_pluginsURL_then_returns_right_URL() {
+        let expectedURL = adminURL + "/plugins.php"
```

Comment:
I'd recommend we use the full absolute URL on each test, that would have caught the double slash I mentioned above.

Reply:
Unfortunately, I think It wouldn't have caught it, because I added the test `adminURL` without a slash:

`let adminURL = "https://testshop.com/wp-admin"`

so the test result only contained one. Now I changed the tests so the given `adminURL` has a slash at the end, same at was is provided by the backend.